tests/dialects/test_snowflake.py

```diff
@@ -1548,6 +1548,29 @@ def test_show(self):
         self.assertTrue(isinstance(users_exp, exp.Show))
         self.assertEqual(users_exp.this, "USERS")
 
+    def test_storage_integration(self):
+        ast = parse_one(
+            """CREATE STORAGE INTEGRATION s3_int
+TYPE=EXTERNAL_STAGE
+STORAGE_PROVIDER='S3'
+STORAGE_AWS_ROLE_ARN='arn:aws:iam::001234567890:role/myrole'
+ENABLED=TRUE
+STORAGE_ALLOWED_LOCATIONS=('s3://mybucket1/path1/', 's3://mybucket2/path2/')""",
+            read="snowflake",
+        )
+        assert isinstance(ast, exp.Create)
```

Comment:
Let's also test that generating `ast` results in the original statement. Other than that, should be good to go.

Reply:
i don’t like this ast test, can we just test identity

Reply:
I can add back the [identity test I had here](https://github.com/tobymao/sqlglot/pull/2985/commits/9ebbfe45ab0d231db3e6f342c875e32680e670eb). I removed it because it passes without my changes, and doesn't test that the statement is parsed as an `exp.Create`

Reply:
We now warn when a statement or expression gets parsed into a `Command`, so as long as there are no warnings when running the test suite, I guess that should be a good-enough signal - at least for these statements where it's easy to see what they're being parsed into.